bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/impl/SingleProbeStrategy.scala

```diff
@@ -255,11 +254,10 @@ object SingleProbeStrategy {
    * Calculate the position angle to a target from a specified base position.
    */
   def calculatePositionAngle(base: Coordinates, st: SiderealTarget): Angle = {
-    val ra1    = st.coordinates.ra.toAngle.toRadians
-    val dec1   = st.coordinates.dec.toAngle.toRadians
-    val target = HmsDegTarget.fromSkyObject(st.toOldModel)
-    val ra2    = Angle.fromDegrees(target.getRa.getAs(Units.DEGREES)).toRadians
-    val dec2   = Angle.fromDegrees(target.getDec.getAs(Units.DEGREES)).toRadians
+    val ra1    = base.ra.toAngle.toRadians
+    val dec1   = base.dec.toAngle.toRadians
+    val ra2    = st.coordinates.ra.toAngle.toRadians
+    val dec2   = st.coordinates.dec.toAngle.toRadians
     val raDiff = ra2 - ra1
```

Comment:
@tpolecat this is currently unused.  It applies when the `PosAngleConstraint` is `UNBOUNDED`, which means AGS is allowed to select any position angle required to reach the highest quality guide star.  Unbounded searches proved problematic, though I can't remember what the issue was.  Maybe someone can refresh my memory?  At any rate, it is a potential task for December to enable this feature so it is good that you caught it.  We should definitely add test cases as part of turning it on.